admin-ui/plugins/auth-server/redux/sagas/AuthnSaga.js

```diff
@@ -12,6 +12,11 @@ import {
 import LdapApi from '../../../services/redux/api/LdapApi'
 import ScriptApi from '../../../admin/redux/api/ScriptApi'
 const JansConfigApi = require('jans_config_api')
```

Comment:
Wrong ScriptApi import path

This path likely doesn’t exist in this repo; the wrapper lives under ../api/ScriptApi.


```diff
-import ScriptApi from '../../../admin/redux/api/ScriptApi'
+import ScriptApi from '../api/ScriptApi'
```